app/boards/shields/quefrency/Kconfig.defconfig

```
@@ -0,0 +1,17 @@
+#Copyright (c) 2020 Noah Thornton
```

Not all the shields are updated yet, but we've been trying to move to a "single author file" approach, and keeping these headers like:

```suggestion
#Copyright (c) 2020 The ZMK Contributors
```

And then of course if you'd like to be added to the `AUTHORS` file, just let us know.